src/main/java/com/massivecraft/factions/cmd/CmdPerm.java

```diff
@@ -34,11 +34,15 @@ public CmdPerm() {
         senderMustBePlayer = true;
         senderMustBeMember = true;
         senderMustBeModerator = false;
-        senderMustBeAdmin = false;
+        senderMustBeAdmin = true;
     }
 
     @Override
     public void perform() {
+        if (!Permission.PERMISSIONS.has(me, true)) {
```

Comment:
unnecessary because setting the perm node in the constructor will make the parent class check for the permission before it ever gets here.

Reply:
If the parent class does that already why are we adding permission checks everywhere else?